Code/GraphMol/Substruct/SubstructMatch.cpp

```diff
@@ -427,6 +427,25 @@ void ResSubstructMatchHelper_(const ResSubstructMatchHelperArgs_ &args,
     delete mol;
   }
 };
+
+struct RecursiveLocker {
+  std::vector<RecursiveStructureQuery *> locked;
+  RecursiveLocker(const ROMol &query, const bool recursionPossible) :
+    locked() {
```

Comment:
I don't think the empty initialization is necessary.
```suggestion
  RecursiveLocker(const ROMol &query, const bool recursionPossible) {
```